src/pylint_datetime/__init__.py

```diff
@@ -1,14 +1,18 @@
-"""Pylint checker ensure correct usage of datetime module, especially around aware/naive objects."""
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Pylint checker ensure correct usage of datetime module, especially around aware/naive objects.
+
+Function calls to fromisoformat, strptime and fromisoformat can't be checked - they parse strings.
+If the specified timezone is None, still allowed although they produce naive objects.
```

Comment:
This should also be in a separate commit, as the module comment is not  related to the removed lines.